pkg/asset/installconfig/azure/metadata.go

```diff
@@ -31,6 +31,10 @@ type Metadata struct {
 	// serve multiple users with different credentials via a web server.
 	Credentials *Credentials `json:"credentials,omitempty"`
 
+	// ManagedBy is a field that can be set as a tag to the azure
+	// resource group.
+	ManagedBy string `json:"managedBy,omitempty"`
```

Comment:
If you want this value to be user-supplied, it should be added to the install config in pkg/types. 

This is adding the field to the install config metadata, I suspect you don't need the field added here.